lib/tower_cli/models/base.py

```diff
@@ -994,7 +994,8 @@ def create(self, fail_on_found=False, force_on_exists=False, **kwargs):
         do not fail (unless `fail_on_found` is set).
         """
         return super(Resource, self).create(
-            fail_on_found=False, force_on_exists=False, **kwargs
```

Comment:
This was an artifact from changing the argument list so-many times. It obviously made the flags stop working.